scalastyle-config.xml

```diff
@@ -0,0 +1,117 @@
+<scalastyle>
```

Comment:
We spoke about this as scalastyle isn't being actively maintained so it might be worth looking for  another (preferably non-xml config) linter.